### ISSUE TITLE
NTR: IL-12 low in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8434]
New term request IL-12 low in blood (elthat_201204131607): IL12; interleukin 12

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000487
- label: Reduced circulating interleukin 12 concentration
- parents: [Abnormal circulating interleukin 12 concentration (HP:6000442)](https://hpo.jax.org/app/browse/term/HP:6000442)
- pmid: 
- def: The concentration of interleukin 12 (IL-12) in the blood circulation is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8434
- synonyms:
